docs/tutorials/google/spin_echoes.ipynb

```diff
@@ -0,0 +1,1362 @@
+{
```

Comment:
<p>The "first" and "second" register are confusing here because we pass <code>[(3, 2), (4, 2), (4, 1), (5, 1), (6, 1)]]</code>  as the qubit order to <code>create_benchmark_circuit</code>  and hence the coupling happens between <code>(3, 2)</code>  and <code>(4, 2)</code>  but in the printed diagram the ordering is sorted lexicographically and hence <code>(3, 2)</code>  is first, <code>(4, 1)</code>  is second and <code>(4, 2)</code>  is third, therefore coupling is b/w 1st and 3rd. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4373/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>